File: ruby/10.rb
def checkPalindromo(palavra)
  reverse = palavra.split('').reverse.join('')

  if reverse == palavra
    puts 'É palíndromo'
  else
    puts 'Não é palíndromo'
  end
end

checkPalindromo('arara') 
checkPalindromo('casa')  

File: js/10.js
const checkPalindromo = (palavra) => {

    const reverse = palavra.split('').reverse().join('')
    if (reverse === palavra) {
        console.log('E palindromo')
        console.log(reverse)
    }

    else {
        console.log('não é')
        console.log(reverse)
    }
}


checkPalindromo('arara')
